Source/Presentation/Transition.swift

```diff
@@ -26,6 +26,10 @@ class Transition: NSObject, UIViewControllerTransitioningDelegate {
         return AnimationController(presentation: true)
     }
 
+    func animationController(forDismissed dismissed: UIViewController) -> UIViewControllerAnimatedTransitioning? {
```

Comment:
I think the Swift migrator messed this up as the method below should have been changed to this. Would you mind deleting that one as part of this PR?